index.html

```diff
@@ -37,86 +36,244 @@
+                        <p> Css </p>
+                    </li>
+                </ul>
             </li>
-            <li class="social-media">
-                <a rel="noopener noreferrer" href="https://twitter.com/SsenkootoSteph1" target="_blank"
-                    aria-label="Twitter"><img src="icons/blacktwitter.svg" alt="Twitter.com Icon"></a>
+            <li>
+                <ul class="container content knowledge">
+                    <li>
+                        <div class="container categories">
+                            <img src="icons/square.svg" alt="">
+                            <h2> Frameworks </h2>
+                        </div>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Bootstrap </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Ruby on Rails </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> RSPec </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> CapyBara </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Selenium </p>
+                    </li>
+                </ul>
             </li>
-            <li class="social-media">
-                <a rel="noopener noreferrer" href="#" aria-label="Medium"><img src="icons/blackM.svg"
-                        alt="Medium.com Icon"></a>
+            <li>
+                <ul class="container content knowledge">
+                    <li>
+                        <div class="container categories">
+                            <img src="icons/circle.svg" alt="">
+                            <h2> Skills </h2>
+                        </div>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Codekit </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Github </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Codepen </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Gitlab </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Terminal </p>
+                    </li>
+                </ul>
             </li>
         </ul>
+    </section>
+
+    <section class="contact_section" id="contact">
+      <h2 class="contact_header">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form
+        class="contact_form"
+        action="https://formspree.io/f/meqnezezgit"
+        method="post"
```

Comment:
- [ ] Good job implementing the action and post for your form 👍 but when I try to submit the form, it says form not found. It would be great if you could address this issue.
![f3](https://user-images.githubusercontent.com/85113567/150104699-939d58d6-2202-4576-aa4d-fc48fe2a1a66.jpg)